content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -5,30 +5,34 @@ weight: 300
 toc: true
 ---
 
-A **resource mapping file** is a JSON file that provides a map of a **lookup name** to a collection of **properties** about an AWS resource. Each mapping has the following format:
+A **resource mapping file** is a JSON file that provides a map of **lookup names** to a collection of **attributes** about AWS resources. O3DE runtimes can query the mapping file to easily find and utilize AWS resources. Usually different deployments, and consequently different mappings, are used for game development and for the released version of a game.
```

Comment:
Update to use active voice, and switch to the more generic "project" instead of "game", per convention in these docs.
```suggestion
A **resource mapping file** is a JSON file that provides a map of **lookup names** to a collection of **attributes** about AWS resources. O3DE runtimes can query the mapping file to easily find and utilize AWS resources. Typically, a project under development will use a different deployment, and therefore a different mapping, than the released version of the project.
```